src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -51,6 +52,14 @@ public void sign(KeyPair signer) {
     mSignatures.add(signer.signDecorated(txHash));
   }
 
+  public static void setDefaultOperationFee(int opFee) {
+    if (opFee < BASE_FEE) {
+      throw new IllegalArgumentException("DefaultOperationFee cannot be smaller than the BASE_FEE (" + BASE_FEE + "): " + opFee);
+    }
+
+    defaultOperationFee = opFee;
+  }
```

Comment:
Wondering if we should move `BASE_FEE`, `defaultOperationFee` and `setDefaultOperationFee` to `Transaction.Builder`. Looks like it's not used in `Transaction` directly.

Reply:
@bartekn transactionBuilder has this already. Instead of setting it on each transactionBuilder instance you can set it as a default for your whole session on Transaction if you want (as a consumer of the sdk). Happy to remove if you feel we should not have this. See the new tests in TransactionTest for examples.

Reply:
My point was to have a `static` field on `Transaction.Builder` instead of `Transaction`.

Reply:
Ah I see — will make the change and update this PR